package.json

```diff
@@ -10,8 +14,10 @@
     "validate-setup": "node scripts/validateSetup.js",
     "test-integration": "sh scripts/tests/testIntegration.sh",
     "test-unit": "sh scripts/tests/testUnit.sh",
-    "test": "npm run lint && npm run test-integration && npm run test-unit",
-    "generate-api-schema": "node utils/generateAPISchema.js"
+    "test": "npm run lint && npm run test-integration && npm run test-unit && bundle exec rake cucumber && bundle exec rake rspec && bundle exec rake jasmine:ci",
+    "generate-api-schema": "node utils/generateAPISchema.js",
+    "requirements-check": "node check_node_version.js",
+    "postinstall": ":.\node_modules/bower/bin/bower install && npm run requirements-check"
```

Comment:
Why are you using bower here?

Reply:
I guess it's used here because after checking the node version, if it's greater than -v14 then we have to install the required dependencies and packages, so it does that only.

Reply:
and for the above issue, all that cucumber jasmine or rspc, I removed that test from the code and committed again

Reply:
@harshith-venkatesh @prakashchoudhary07 Please review it again, I have made the changes suggested

Reply:
Please recheck, we are not using bower as a package manager here

Reply:
Yes, we are using npm right ?? It is a silly mistake.

Reply:
@prakashchoudhary07 Changed the package manager from bower to npm. Please review